code/modules/mob/living/carbon/human/species_types/mothmen.dm

```diff
@@ -57,7 +65,7 @@
 
 /datum/species/moth/space_move(mob/living/carbon/human/H)
 	. = ..()
-	if(H.loc && !isspaceturf(H.loc) && H.dna.features["moth_wings"] != "Burnt Off")
+	if(H.loc && !isspaceturf(H.loc) && H.dna.features["moth_wings"] != "Burnt Off" && H.dna.features["wings"] != wings_icon ) //"wings" are exclusive to the potion of flight, and those have their own flying mechanics. If they want to fly they can use that mechanic instead
```

Comment:
Why not use `flying_species` here as well?

Reply:
I don't think I considered checking that var, and I'm too drunk presently to even attempt to remember that properly, but I can certainly look into attempting to use that instead of checking what they have for wings when I'm more coherent.

Reply:
Check it when you are sober then.